webgl/lessons/webgl-2d-matrices.md

```diff
@@ -364,10 +364,10 @@ We also removed the code that set the resolution. With this last step we've gone
 I hope these posts have helped demystify matrix math. <a href="webgl-3d-orthographic.html">I'll move on to 3D next</a>. In 3D matrix math follows the same principles and usage. I started with 2D to hopefully keep it simple to understand.
 
 <div class="webgl_bottombar">
-<h3>What are <code>clientWidth</code> and <code>clientHeight</code>?</h3>
```

Comment:
markdown doesn't parse inside HTML. The fact that all of this is wrapped with `<div class="webgl_bottombar">` means everything inside is just copied verbatim. That means your backticks won't get converted into `<code>` blocks.